src/main/kotlin/ink/abb/pogo/scraper/tasks/CatchOneNearbyPokemon.kt

```diff
@@ -46,7 +46,7 @@ class CatchOneNearbyPokemon : Task {
 
         if (pokemon.isNotEmpty()) {
             val catchablePokemon = pokemon.first()
-            if (settings.obligatoryTransfer.contains(catchablePokemon.pokemonId) && settings.desiredCatchProbabilityUnwanted == -1.0) {
+            if (settings.obligatoryTransfer.contains(catchablePokemon.pokemonId) && settings.desiredCatchProbabilityUnwanted == -1.0 || settings.neverCatchPokemon.contains(catchablePokemon.pokemonId)) {
```

Comment:
those settings are not mine, however there is && between "obligatory transfer" and "desiredcatchprobability unwanted". This is from someone else and means, if the catchprobability of a pokemon is -1.0 dont catch it, and the desired_catch_probability_unwanted is meant for pokemon in the obligatory transfer list